### PR TITLE
[Storybook][File Sharing] Update file sharing for Teams Interop scenario

### PR DESCRIPTION
**Pull request by AmyL219**
# What

# Why

# How Tested

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx [hunk 1]
```diff
@@ -76,3 +90,44 @@ export const getDocs: () => JSX.Element = () => {
...
+  return (
+    <>
+      <Title>File Sharing</Title>
+      <SingleLineBetaBanner />
+      <Description>Microsoft Teams enables file attachment sharing with the following permissions:</Description>
+      <Description>- Anyone</Description>
+      <Description>- Individuals within your organization</Description>
+      <Description>- Participants currently in this chat</Description>
+      <Description>- People who already have access</Description>
+      <Description>- Specific individuals you select</Description>
```
Comment (jimchou-dev) [inline packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx:104]: Can we add an example screenshot here?

File: packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx [hunk 2]
```diff
@@ -76,3 +90,44 @@ export const getDocs: () => JSX.Element = () => {
...
+      <Description>
+        Currently, Azure Communication Services only supports the "Anyone" and "People you choose" options. The “People
```
Comment (jimchou-dev) [inline packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx:106]: Can we add a line something along the lines of `Microsoft Teams allows user to share file attachments as part of chat message, these files are stored in SharePoint or OneDrive that require authentication to access those files.`

File: packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx [hunk 3]
```diff
@@ -76,3 +90,44 @@ export const getDocs: () => JSX.Element = () => {
...
+        in your organization”, “People currently in this chat”, and “People with existing access” permissions can only
+        be supported for users in a Teams tenant. To learn more about the Sharing Permission, please refer to the [Limit
+        sharing in Microsoft
+        365](https://learn.microsoft.com/en-us/microsoft-365/solutions/microsoft-365-limit-sharing?view=o365-worldwide).
```
Comment (jimchou-dev) [inline packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx:110]: Strip out locale (will auto redirect based on browser language), and for some reason MSDocs PR ask to remove query uri parameter:
`https://learn.microsoft.com/microsoft-365/solutions/microsoft-365-limit-sharing`

**Review by jimchou-dev — COMMENTED** [review]
Nice refactoring to make things more clear

**Review by jimchou-dev — APPROVED** [review]
LGTM

File: packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx [hunk 4]
```diff
@@ -124,7 +124,8 @@ export const getDocs: () => JSX.Element = () => {
         solution. Using the provided APIs, you can enable the composite to support uploading files and displaying them
         on the send box before sending, and the message thread once they have been sent or received. For an end to end
         tutorial on enabling file sharing with Azure Blob Storage as your choice of a storage solution, refer to our
-        tutorial. https://docs.microsoft.com/en-us/azure/communication-services/tutorials/file-sharing-tutorial
+        [tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/file-sharing-tutorial). File
+        sharing is supported for Teams interop scenarios by default.
```
Comment (JoshuaLai) [inline packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx:128]: Should we be specific here and call out: File sharing from Teams to ACS is supported. And when the next feature is released just update it?

**Review by JoshuaLai — APPROVED** [review]
All the changes look good. Just think we should be a little more specific with the messaging saying file sharing only works one way for now from Teams -> ACS

**Comment by AmyL219** [comment]
> All the changes look good. Just think we should be a little more specific with the messaging saying file sharing only works one way for now from Teams -> ACS

Yeah, we have mentioned one sentence there:      
```
 <Description>
        At this time, the UI library only facilitates File Sharing in a Teams Interop chat sent by the Teams user. To
        utilize this feature, you can employ the `CallWithChat` composite.
</Description>
```

**Pull request merged**